tests/utils.py

```diff
@@ -247,9 +247,8 @@ def skip_if_not_active(destination: str) -> None:
 
 def is_running_in_github_fork() -> bool:
     is_github_actions = os.environ.get("GITHUB_ACTIONS") == "true"
-    head_ref = os.environ.get("GITHUB_HEAD_REF", "")
-    repo = os.environ.get("GITHUB_REPOSITORY", "")
-    return is_github_actions and ":" in head_ref and not head_ref.startswith(repo.split("/")[0])
```

Comment:
These variables are not very reliable (like any paths on GitHub):

![image](https://github.com/dlt-hub/dlt/assets/26030071/121b3c55-5e56-4bdd-9989-b86ac4b6e998)